src/mame/nintendo/famibox.cpp

```diff
@@ -396,7 +382,7 @@ void famibox_state::famibox_map(address_map &map)
 {
 	map(0x0000, 0x1fff).ram();
 	map(0x2000, 0x3fff).rw(m_ppu, FUNC(ppu2c0x_device::read), FUNC(ppu2c0x_device::write));
-	map(0x4014, 0x4014).w(FUNC(famibox_state::sprite_dma_w));
+	map(0x4014, 0x4014).lw8(NAME([this] (address_space &space, u8 data) { m_ppu->spriteram_dma(space, data); }));
```

Comment:
This changes behaviour, getting rid of the `& 7` mask.  But that aside, same question as with the other case – is there a reason it can’t be installed directly?

Reply:
``data`` here should be any page of memory. The only thing I could make of those masks is that they originated from a misunderstanding that DMA can only copy from CPU main memory. Such a limitation doesn't exist, but if you're just looking at what most games do in practice maybe you'd make that guess. It just looks like the with and without mask versions were copy pasted haphazardly over time.